composer.json

```diff
@@ -24,7 +24,7 @@
         "drupal/config_split": "~1.3",
         "drupal/config_update": "~1.2",
         "drupal/context": "~4.0@beta",
-        "drupal/core": "^8.6.0",
+        "drupal/core": "8.6.*",
```

Comment:
@jeremy-doghouse you could even use the tilde as well https://getcomposer.org/doc/articles/versions.md#tilde-version-range-